src/components/Reports/SalaryReport/SalaryReportTable.tsx

```diff
@@ -1,10 +1,219 @@
-import React from 'react';
-import { Box } from '@material-ui/core';
+/* eslint-disable import/no-unresolved */
+import React, { useMemo } from 'react';
+import clsx from 'clsx';
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  CircularProgress,
+  Grid,
+  styled,
+  SvgIcon,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import CodeIcon from '@material-ui/icons/Code';
+import PrintIcon from '@material-ui/icons/Print';
+import { CSVLink } from 'react-csv';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import { FourteenMonthReportCurrencyType } from '../../../../graphql/types.generated';
+// eslint-disable-next-line import/extensions
+import { useFourteenMonthReportQuery } from 'pages/accountLists/[accountListId]/reports/graphql/GetReportFourteenMonth.generated';
+import { Notification } from 'src/components/Notification/Notification';
+import { EmptyReport } from 'src/components/Reports/EmptyReport/EmptyReport';
 
 interface Props {
+  className?: string;
   accountListId: string;
+  title: string;
 }
 
-export const SalaryReportTable: React.FC<Props> = () => {
-  return <Box />;
+const useStyles = makeStyles(() => ({
+  root: {},
+}));
+
+const DownloadCsvLink = styled(CSVLink)(({}) => ({
+  color: 'inherit',
+  textDecoration: 'none',
+}));
+
+const YearTableCell = styled(TableCell)(({}) => ({
+  paddingLeft: 0,
+}));
+
+const YearTypography = styled(Typography)(({ theme }) => ({
+  borderLeft: `1px solid ${theme.palette.cruGrayLight.main}`,
+}));
+
+export const SalaryReportTable: React.FC<Props> = ({
+  className,
+  accountListId,
+  title,
+  ...rest
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const { data, loading, error } = useFourteenMonthReportQuery({
```

Comment:
Since this is where the query is happening, can you move the `.graphql` file to live in the same folder as this file? Unless it's being shared or something.